src/api/news.js

```diff
@@ -1,10 +1,31 @@
-import request from '@/utils/request'
-import config from './config'
+import APIBase from '.'
 
-export function getNews(data) {
-  return request({
-    url: config.baseURL + `/news`,
-    method: 'get',
-    params: data
-  })
+class NewsAPI extends APIBase {
+  constructor() {
+    super()
+    this.prefix_path = '/news'
+  }
+
+  /**
+   * search news in db
+   * @param {String} keyword : keyword to search
+   * @param {Number} pageNum : page number
+   * @param {String} tbs : time range
+   * @param {String} mediaSource : media source
+   * @returns {Promise<Any>}
+   */
+  search(keyword = '', pageNum = 0, tbs = 'qdr:a', mediaSource = '') {
```

Comment:
You can use search(params) and `const { keyword: q, pageNum: page, tbs, mediaSource: media } = params`
instaed of 4 parameters function, and not sending `mediaSource = ''` to BE